dockerizeit/master/permissions.properties

```diff
@@ -0,0 +1,12 @@
+user1 {
+		userId = hudson.security.ACL.ANONYMOUS_USERNAME
+		permissions = [
+			"hudson.model.Computer.Create",
```

Comment:
Don't use strings - use actual classes, i.e. remove quotes then you don't need to do Permission.fromId(it) - you could simply say - it. But make sure to import 'import hudson.security.Permission' and 'import hudson.model.Computer' in the properties file